pyteal/ast/unaryexpr_test.py

```diff
@@ -357,6 +358,27 @@ def test_b_not_invalid():
         BytesNot(Int(2))
 
 
+def test_bsqrt():
+    arg = Bytes("base16", "0xFEDCBA9876543210")
+    expr = BytesSqrt(arg)
+    assert expr.type_of() == TealType.bytes
+
+    expected = TealSimpleBlock(
+        [TealOp(arg, Op.byte, "0xFEDCBA9876543210"), TealOp(expr, Op.bsqrt)]
+    )
+
+    actual, _ = expr.__teal__(teal6Options)
+    actual.addIncoming()
+    actual = TealBlock.NormalizeBlocks(actual)
+
+    assert actual == expected
+
+
+def test_bsqrt_invalid():
+    with pytest.raises(TealTypeError):
+        BytesNot(Int(2 ** 64 - 1))
```

Comment:
```suggestion
        BytesSqrt(Int(2 ** 64 - 1))
```

This should be `BytesSqrt`

Reply:
Whoops